Exiled.API/Features/Player.cs

```diff
@@ -959,7 +959,7 @@ public bool BadgeHidden
         /// <summary>
         /// Gets a value indicating whether or not the player is in the pocket dimension.
         /// </summary>
-        public bool IsInPocketDimension => IsEffectActive<Corroding>() || Map.FindParentRoom(GameObject)?.Type == RoomType.Pocket;
+        public bool IsInPocketDimension => IsEffectActive<Corroding>() && Map.FindParentRoom(GameObject)?.Type == RoomType.Pocket;
```

Comment:
```suggestion
        public bool IsInPocketDimension => Map.FindParentRoom(GameObject)?.Type == RoomType.Pocket;
```

I don't see the need to make a check on that effect, considering there isn't much need, and player could still be in pocket dimension without having the effect

Reply:
i thinks it's more about getting player getting normally in this room so people without effect like Teleporting player will not be considering in the Room could be good for some plugin to have this

Reply:
My point was plugin could teleport the player to the pocket dimension and they won't have the effect.

Reply:
This should return true if they are physically in the pocket dimension, regardless of whether they have the effect.